### PR TITLE
#1 Fix chapter01/SimpleEvenOddFuncMain.kt : The value of 9 is unused

### PR DESCRIPTION
**Pull request by LeoAndo**
Problems and measures: See #1

### CODE REVIEW

**Comment by LeoAndo** [comment]
@supriyathabe @RivuChk

Hello. I am Japanese engineer. Code review please.

File: Chapter01/src/main/kotlin/com/rivuchk/packtpub/reactivekotlin/chapter01/IteratorExmpl.kt [hunk 1]
```diff
@@ -6,8 +6,8 @@ package com.rivuchk.packtpub.reactivekotlin.chapter01
  */
 
 fun main(args: Array<String>) {
-    var list:List<Any> = listOf("One", 2, "Three", "Four", 4.5, "Five", 6.0f) // 1
-    var iterator = list.iterator() // 2
+    val list:List<Any> = listOf("One", 2, "Three", "Four", 4.5, "Five", 6.0f) // 1
+    val iterator = list.iterator() // 2
```
Comment (LeoAndo) [inline Chapter01/src/main/kotlin/com/rivuchk/packtpub/reactivekotlin/chapter01/IteratorExmpl.kt:10]: Change the write protection variable to val.

File: Chapter02/src/main/kotlin/com/rivuchk/packtpub/reactivekotlin/chapter02/CoroutineExample1.kt [hunk 2]
```diff
@@ -1,7 +1,5 @@
 package com.rivuchk.packtpub.reactivekotlin.chapter02
 
-import kotlinx.coroutines.experimental.CommonPool
-import kotlinx.coroutines.experimental.async
```
Comment (LeoAndo) [inline Chapter02/src/main/kotlin/com/rivuchk/packtpub/reactivekotlin/chapter02/CoroutineExample1.kt:4]: Delete unnecessary import statements.

File: Chapter07/src/main/kotlin/com/rivuchk/packtpub/reactivekotlin/chapter07/chapter7_1_UsualExample.kt [hunk 3]
```diff
@@ -1,11 +1,8 @@
-package com.rivuchk.packtpub.reactivekotlin.chapter7
+package com.rivuchk.packtpub.reactivekotlin.chapter07
```
Comment (LeoAndo) [inline Chapter07/src/main/kotlin/com/rivuchk/packtpub/reactivekotlin/chapter07/chapter7_1_UsualExample.kt:1]: Changed because the package name is invalid.

**Comment by LeoAndo** [comment]
@supriyathabe @RivuChk

It was a wonderful book. I learned a lot.

https://www.amazon.co.jp/Reactive-Programming-Kotlin-non-blocking-Reactor-Kotlin/dp/1788473027

**Comment by RivuChk** [comment]
Thanks @LeoAndo for submitting PR and for these generous comments on PR. Merged.

**Pull request merged**